README.md

```diff
@@ -91,7 +91,7 @@ The automated `STEPS`:
 
 The following credentials are needed
 
-* Github token: as environment variable `GITHUB_TOKEN` or `--githubtoken file`.
+* Github token: as environment variable `GITHUB_TOKEN`, `GH_TOKEN`, or `--githubtoken file`.
```

Comment:
I'm a little concerned here in that past experience has these being two different tokens and I'm not sure they should be reused. I know one can authenticate `gh` using a GITHIB_TOKEN, but does it work the other way?